docs/testplan/account-changes-testplan.md

```diff
@@ -0,0 +1,33 @@
+| New user on Storj.                   | 1. Default parameters.                                             | Free trial duration - 30 days. Storage -25 GB. Egress - 25 GB                                                                                                                                                                                                                                                                                |          |
+|                                      | 2. Coupon absence.                                                 | Coupon is not applied. No Coupon placeholder on the Billing page                                                                                                                                                                                                                                                                               |          |
+| Already existed user.                | 3. Existed user impact.                                            | For current customers, the free tier  will be sunsetting. We will email all users who are on the free trial four times: A 30-day notice, a 15-day notice, and a final notice on the final day.                                                                                                                                                                                                                                                                                                              |          |
+| Trial period configuration.          | 4. Trial period configuration - period changes.                    | Only new users should be affected.                                                                                                                                                                                                                                                                                                              |          |
+|                                      | 5. Trial period limits changes.                                    | Only new users should be affected.                                                                                                                                                                                                                                                                                      |          |
+| Migration.                           | 6. Migration command.                                              | Command should affect Free tier users. Migrated users should have 0 limits, and a “free trial expiration” event in the DB.                                                                                                                                                                                 |          |
```

Comment:
this will actually be different
migration is run with an expiration date argument
after it is run, migrated users should have `trial_expiration=<date>`, which can be in the future. They will not have "free trial expiration" or 0 limits until after that date passes and the freeze chore executes.

There is also another part of the migration which will be run first: migrate free tier users who have legacy STORJ token payments to the paid tier - that way they are not affected by the main migration

Reply:
changed